andela/strikers-ah-backend
# #164489787 Rate article

#### What does this PR do?
This helps the user to rate an article with **Terrible, Bad, Okay, Good, or Great**.
#### Description of Task to be completed?
- Created the request route and its handler/controller
- Tested the feature
- Document it using swagger and jsDocs
#### How should this be manually tested?
- After cloning the repository with https://github.com/andela/strikers-ah-backend`, check out this branch `ft-rate-article-164489787`
- Run the dependencies installation with `npm install`
- Start the server with `npm run dev`.
- Access the below endpoint with any HTTP Client application (usually: POSTMAN).
 `api/articles/<articleSlug>/<rate>` WHERE `rating` is one of **Terrible, Bad, Okay, Good, or Great**
#### What are the relevant pivotal tracker stories?
#164489787
#### Questions:
none
